css/soe_helper.css

```diff
@@ -1738,6 +1738,10 @@ body:not(.page-admin) .sidebar .views-exposed-form .views-exposed-widget .form-s
       font-size: 0.9em;
       line-height: 25px; }
 
+/* line 118, ../scss/components/_soe_events.scss */
```

Comment:
How do we use these comments?

Reply:
Those comments are used by your browser to tell your inspector where the css came from.

Reply:
Oh, that's awesome!

Reply:
this information is there so that when you are viewing the css that gets loaded on a page, you can tell where in the scss it was generated.